fxapom/pages/sign_in.py

```diff
@@ -64,6 +64,9 @@ def check_for_popup(self, handles):
                     break
             else:
                 raise Exception('Popup has not loaded')
+        else:
+            Wait(self.driver, self.timeout).until(
+                EC.visibility_of_element_located(self._email_input_locator))
```

Comment:
While this looks like it would work, it does introduce duplication. Could you experiment with moving the wait from lines 61-62 down to here? It doesn't need to be within an `else` block as this should only be reached if the appropriate popup window has been found or if there are no popups present.